addon/components/flexberry-map.js

```diff
@@ -77,6 +90,7 @@ export default Ember.Component.extend(
       this.set('_layer', map);
       this._addObservers();
       this._addEventListeners();
+      this.sendAction('leafletMapDidInit', map);
```

Comment:
We don't have any other maps except leaflet inside component.
So such looong action/properties names are unnecessary.